wp-shopify.php

```diff
@@ -26,4 +26,132 @@ function pp() {
         }
     }
 
+    // Set rewrite slug on first activation
+    function set_wps_rewrite_slug(){
+        if( get_option('wshop_rewrite_slug') == '' ){
+            update_option( 'wshop_rewrite_slug', 'store' );
+            flush_rewrite_rules();
+        }
+    }
+
+    register_activation_hook( __FILE__, 'set_wps_rewrite_slug' );
+
+    // Define AJAX functions
+    function wps_process_product(){
+
+        // Get the ID of the current Product
+        $id = $_REQUEST['product_id'];
+        // Get the title of the current Product
+        $title = $_REQUEST['product_title'];
+        // Get refresh options
+        $auto_publish = $_REQUEST['auto_publish'] == 'true';
+        $auto_delete = $_REQUEST['auto_delete'] == 'true';
+
+        $output = 'Error processing Product ' . $title . '!';
+
+        // Find any existing Products that match the desired ID
+        $args = array(
+            'posts_per_page'    => -1,
+            'post_type'         => 'wps-product',
+            'meta_key'          => '_wshop_product_id',
+            'meta_value'        => $id,
+            'post_status'       => 'publish,private,draft,future,pending'
```

Comment:
@jrobson153 I didn't want to look in the trash - should it still be set to 'any'?